util/dvsim/SimCfg.py

```diff
@@ -9,11 +9,10 @@
 import sys
 from collections import OrderedDict
 
-from testplanner import class_defs, testplan_utils
-
 from Deploy import *
 from FlowCfg import FlowCfg
 from Modes import *
+from testplanner import class_defs, testplan_utils
```

Comment:
If this change came from a rebase, it should not have showed up, right?

Reply:
not from rebase. It is on line 12 in latest version and lintpy moved it

Reply:
ohh, i see. yapf is super flaky.